build.gradle.kts

```diff
@@ -690,8 +690,12 @@ fun printDockerCheckInfo() {
   val dockerRunning = project.extra["dockerRunning"] as? Boolean ?: false
   val macDockerConnector = project.extra["macDockerConnector"] as? Boolean ?: false
   val isOrbStack = project.extra["isOrbStack"] as? Boolean ?: false
-
-  if (extra["skipDockerTests"].toString().toBoolean()) {
+  val skipDockerTest = if (extra["skipDockerTests"].toString().toBoolean()) {
+    System.getenv("SKIP_DOCKER_TESTS")?.toBoolean() ?: true
+  } else {
+    false
+  }
```

Comment:
Please add comments to clarify the logic.

Reply:
I have add some comments, hope it could help to make the logic clearer.